Project.toml

```
@@ -40,6 +42,7 @@ Graphs = "1.6"
 KaHyPar = "0.3.1"
 Makie = "0.19, 0.20, 0.21, 0.22"
 PackageExtensionCompat = "1"
+SparseArrays = "1.11.0"
```

can this be lowered to 1.10?

```suggestion
SparseArrays = "1.10.0"
```